R/io.R

```diff
@@ -562,9 +562,11 @@ download_civis.numeric <- function(x, file,
   }
 
   args <- c(list(files_get(x)$fileUrl),
-            list(httr::write_disk(file, overwrite = overwrite)))
+            list(httr::write_disk(file, overwrite = overwrite)),
+            verb = "GET",
+            terminate_on = c(403, 404))
```

Comment:
Could you also please make this `terminate_on = setdiff(400:499, 429)`?  Sorry, I should have commented about each case rather than just the first.

Reply:
oh sure, no problem!